altair/__init__.py

```diff
@@ -1,6 +1,610 @@
 # flake8: noqa
+import sys
```

Comment:
Why do we need to import sys here? It seems like it is not used in this file.

Reply:
Indeed, removed it. That was from a previous implementation.